docs/IOS_FASTLANE_SCREENSHOTS_ESCORT.md

```
@@ -0,0 +1,244 @@
+### 2.5 実行中の流れ
+
+1. **シミュレーターが起動**（iPhone 16 Pro Max 等）
+2. **アプリがビルド・起動**
+3. **SnapshotUITests** のテストが実行され、各画面でスクリーンショットを撮影
+4. 完了後、**`fastlane/screenshots/`** に画像が保存されます
+
+---
+
+### 2.6 出力先
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)